hw_diag/tasks.py

```diff
@@ -39,7 +40,12 @@ def perform_hw_diagnostics(ship=False):  # noqa: C901
     diagnostics['AN'] = public_keys['name']
 
     # Fetch data from miner container.
-    diagnostics = fetch_miner_data(diagnostics)
+    try:
+        diagnostics = fetch_miner_data(diagnostics)
+    except MinerFailedFetchData as e:
```

Comment:
I was thinking about leaving it just in case we need it in the future, I know it won't be called since it's no longer being raised in `fetch_miner_data` but you're right, it's better to remove it rather than leave unwanted code that may cause confusion.